.github/actions/filter-changes/action.yml

```
@@ -0,0 +1,41 @@
+name: "Filter Changes"
+description: "Filters changed files based on 'every' and 'any' file patterns"
+
+inputs:
+  every_filter:
+    description: "Patterns where every file must match (in YAML array format)"
+    required: true
+  any_filter:
+    description: "Patterns where any file can match (in YAML array format)"
+    required: true
```

In most of the places we use dashes instead of underscores in input and output parameter names.

```suggestion
  every-filter:
    description: "Patterns where every file must match (in YAML array format)"
    required: true
  any-filter:
    description: "Patterns where any file can match (in YAML array format)"
    required: true
```

---

@scheibinger sure, let me change it. Btw, I think underscores work better with yml. I saw some issues in the past with dashes

---

fixed in last commit